docs/blog/introducing-modern-web/index.md

```diff
@@ -255,7 +255,108 @@ If you wanna know more like for example how to test css media queries see the [r
 
 ## Using Code Coverage
 
-...
+Once you have a decent set of tests you may want to look into what could still be improved.
+Code coverage can help to find which code segments have not yet been tested.
+Generally it's advised to have a code coverage above 80% which you will definitely have if you do test driven development.
+
+## Getting the test coverage
+
+Coverage is part of the default feature set that comes with any launcher that works with chromium.
```

Comment:
maybe specify "any web-test-runner launcher that works with chromium" or something like that?

Reply:
changed